src/utils/sformat.fs

```diff
@@ -54,65 +54,73 @@ namespace Microsoft.FSharp.Text.StructuredFormat
 
     [<NoEquality; NoComparison>]
 #if COMPILER
+    type internal BoxRange = BoxRange of obj
```

Comment:
Sure, I understand the rationale, but it's easy to move the file to be on its own. There's no need for us to hack around like this. Just move the file up to the top of the project - like you said it's only a small type.